README.md

```diff
@@ -144,3 +144,13 @@ julia> hash(Box2{Int}(1))
 julia> hash(Box2{Any}(1))
 0xefe691a94f296c61
 ```
+
+## Specifying the "type seed"
+
+When we compute the hash function, we start with a "seed" particular to the type being hashed.  You can select the seed to be used by specifying `typeseed=e`.
+
+The seed provided (`e`) is used in one of two ways, depending on the setting for `typearg`.
+If `typearg=false` (the default), then `e` will be used as the type seed.
+If `typearg=true`, then `e(t)` is used as the type seed, where `t` is the type of the object being hashed.
+
+Note that the value of `typearg` is a value when `typearg=false` (or `typearg` is not specified), but a function when `typearg=true`.
```

Comment:
```suggestion
Note that the value of `typearg` is a value when `typearg=false` (or `typearg` is not specified), but a function that takes a type as its argument when `typearg=true`.
```

Reply:
maybe add a sentence on why one is a value and one is a function as it's not the most obvious

Reply:
Also the first occurrence in this line of "typearg" should be "typeseed"